release-notes/topics/aap-211.adoc

```diff
@@ -0,0 +1,16 @@
+[[aap-2.1.1-intro]]
+= Ansible Automation Platform 2.1.1
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements and bug fixes
+
+* Added OCP 4.10 compatibility
+* Added support for HTTP proxy when pushing execution environment images
+* Added changes to the billing operator to be cluster-scoped
+* Added ability to configure nginx headers on Controller
+* Changed the setup.log file to no longer be world readable
+* Fixed an imagePullBackoff error when pulling execution environment images during a Controller install on OCP
+* Fixed a bug where the installer loads incorrect bundles instead of the proper ee-supported-rhel8:latest bundle
```

Comment:
Which JIRA are you refering to here ?

Reply:
This one: https://issues.redhat.com/browse/AAP-1935

Reply:
This can be removed. It wasn't a 2.1.1 fix. It was a build issue when we created 2.1.1-1 that was fixed in 2.1.1-2 (just a rebuild)